python-exercise-01/app/calculator.py

```diff
@@ -0,0 +1,18 @@
+
+
+def calculator ( number1, number2, operation ):
+
+    operations = [ "+", "-", "/", "*" ]
+
+    results = {
+        '+': number1 + number2,
+        '-': number1 - number2,
+        '/': number1 / number2,
+        '*': number1 * number2,    
+    }
+
+    if operations.__contains__(operation):
```

Comment:
Esta linea hace referencia metodos internos, lo cual no es recomendable, se recomienda para implementar codigo python mas alineado con las buenas pracitacs y la filosofia implementarla asi:
```python
if operation in operations:
```
Es mas facil de leer, mas alla que el `in` ejecuta la ejecute el metodo `__contains__`.